packages/select/Select.tsx

```diff
@@ -0,0 +1,116 @@
+import { h, Component } from 'skatejs';
+import { GenericEvents, prop } from '../_helpers';
+import Option from './Option';
```

Comment:
I have some problems with Option import from ./index, I will talk to you later...